cmd/batch-expire.go

```diff
@@ -286,24 +287,31 @@ type BatchJobExpire struct {
 	Rules           []BatchJobExpireFilter `yaml:"rules" json:"rules"`
 }
 
-var _ yaml.Unmarshaler = &BatchJobExpire{}
+// BatchJobExpireV2 represents configuration parameters for a batch expiration
+// job typically supplied in yaml form
+type BatchJobExpireV2 struct {
```

Comment:
You can keep the struct as BatchJobExpire and move the existing one into BatchJobExpireV1